pyogrio/raw.py

```diff
@@ -150,12 +153,12 @@ def write(
     layer=None,
     driver=None,
     # derived from meta if roundtrip
-    geometry_type=None,
+    geometry_type: Optional[str] = None,
```

Comment:
```suggestion
    geometry_type=None,
```

Per comments on #74, I think if we are going to add type annotations, we should do so as a comprehensive effort, not piecemeal.  Please leave out here.